src/thunderbots/software/util/parameter/config/ai.yaml

```diff
@@ -46,7 +46,7 @@ AI:
     ideal_min_rotation_to_shoot_degrees:
       min: -180
       max: 180
-      default: 5
+      default: 60
```

Comment:
* This should be a positive range since it should be for the absolute angle difference (so min should be 0)
* A better name might be `max_idea_rotation_to_shoot_degrees`, since it doesn't make sense to define a min. We can assume the min of 0 is ideal, so we only need to specify an upper bound
* The description should also be updated accordingly after the above changes.

Reply:
Changed